<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>3</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(136, 138, 133);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1000</width>
      <height>311</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>311</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>761</width>
      <height>311</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resource/ombori_white4.png</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>702</width>
      <height>498</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="TouchScreenTest_Button">
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 52, 54);</string>
       </property>
       <property name="text">
        <string>Touch Screen 
Test&amp;&amp;Calibrate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PrinterTest_button">
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 52, 54);</string>
       </property>
       <property name="text">
        <string>Printer Test</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="networkTest_Button">
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 52, 54);</string>
       </property>
       <property name="text">
        <string>Network Test</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BarcodeScanner_button">
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 52, 54);</string>
       </property>
       <property name="text">
        <string>Barcode Scanner Test</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="poweroff_Button">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(46, 52, 54);</string>
         </property>
         <property name="text">
          <string>Poweroff for
GridOS</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_Button">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(46, 52, 54);</string>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
